portal/src/graphql/portal/Web3ConfigurationAddCollectionSection.tsx

```diff
@@ -222,6 +234,10 @@ const Web3ConfigurationAddCollectionForm: React.VFC<AddCollectionSectionProps> =
           onContinue={handleAddCollection}
           onDismiss={dismissDialogs}
         />
+        <Web3ConfigurationContactUsDialog
+          isVisible={!contactUsDialogHidden}
+          onDismiss={onDismissContactUsDialog}
+        />
```

Comment:
One minor ui comment. The `form` element is `display:flex` with `gap`. Maybe we should put the dialog outside the `form`. 

The problem is when the dialog is shown, 1 more element is added to the `form` which makes the form move.

Before the dialog is shown:
<img width="757" alt="Screenshot 2022-10-31 at 10 57 41 am" src="https://user-images.githubusercontent.com/388892/198993606-594b06a4-e196-4708-96f1-094d3aafe767.png">

After the dialog is shown:
<img width="758" alt="Screenshot 2022-10-31 at 10 57 33 am" src="https://user-images.githubusercontent.com/388892/198993620-8afc6565-45cd-4a65-bede-729d2570d7ae.png">